curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -1152,16 +1242,19 @@ ChunkCacheManager::CreateWriteDataCache(S3ClientAdaptorImpl *s3ClientAdaptor,
                                         const char *data) {
     DataCachePtr dataCache =
         std::make_shared<DataCache>(s3ClientAdaptor, this, chunkPos, len, data);
-    VLOG(9) << "CreateWriteDataCache chunkPos:" << chunkPos << ", len:" << len;
+    VLOG(9) << "CreateWriteDataCache chunkPos:" << chunkPos << ", len:" << len
```

Comment:
it seams current function is no need to return a value

Reply:
why？

Reply:
because no one handles this return value. but, now, I think you should rename this function to WriteNewDataCache, because you just create a new data cache and copy write data to it.